modules/quadkey/bigquery/sql/_BBOX_E.sql

```diff
@@ -0,0 +1,8 @@
+----------------------------
+-- Copyright (C) 2021 CARTO
+----------------------------
+
+CREATE OR REPLACE FUNCTION `@@BQ_PREFIX@@quadkey._BBOX_E`(quadint INT64)
```

Comment:
There is a missing '_',  at the start of every private function, normally use double '_' in BQ.
Also normally each .sql file refers to a public function and the auxiliar/private functions are inside the same file. Maybe you can add them inside the BBOX one. Or at least gather them into less files as __BBOX_N,__BBOX_S... and __TILE2LAT, __TILE2LON are related.